### DISCUSSION TITLE
Is there a way to break line in remain-on-exit-format?

### DISCUSSION THREAD

**Discussion by raphaeltannous** [tmux/tmux#4198] [General]
Hey,

I am using tmux 3.5a, while I was scripting an ssh manager script for tmux. I wanted to be able to add more information in `remain-on-exit-format` on another line.

For example:
```sh
tmux set-option -t $SESSION:"$window_name" remain-on-exit-format "Pane is dead, (Exit status: #{pane_dead_status}, #{t:window_activity}). Use respawn-window to restart ssh connection to $1."

# Wanted output:
# Pane is dead, (Exit status: #{pane_dead_status}, #{t:window_activity})
# Use respawn-window to restart ssh connection to $1.
```

I tried using `\n` and tried to break the string using `\`. I also tried `#(echo)` and `#(printf "\n")`.

I also tried using `remain-on-exit-format[0]`...

Is there a way to break a line in only `remain-on-exit-format`, because I don't think it is necessary in other place.

Thanks in advance.

**Comment by nicm** [comment]
It only draws one line right now, you can't have more than one.

**Reply by raphaeltannous** [reply]
Okay thank you!
